components/layout_2020/style_ext.rs

```diff
@@ -361,6 +364,14 @@ impl ComputedValuesExt for ComputedValues {
         // TODO: We need to handle CSS Contain here.
         false
     }
+
+    fn get_webrender_primitive_flags(&self) -> wr::PrimitiveFlags {
+        if self.get_box().backface_visibility == BackfaceVisiblity::Visible {
+            wr::PrimitiveFlags::default()
+        } else {
+            wr::PrimitiveFlags::empty()
+        }
```

Comment:
Nit: please use `match` instead of `==` so that the compiler reminds us to update this code in case a new variant is ever added to the enum. (I suspect it is very unlikely that the `backface-visibility` property will ever get a new value, but making more common a pattern that can help elsewhere seems good.)

Reply:
This is also fewer lines of code, so it makes a lot of sense.